problems/removeVowels.js

```diff
@@ -6,8 +6,24 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+const removeVowels = (str) => {
+    let newStr = ""
+    for(let i =0; i < str.length;i++){
```

Comment:
Iterating with a for of loop can be helpful with strings when we don't care about indicies. It will lead to cleaner code.